feeds/mountaintransit.org.dmfr.json

```diff
@@ -5,9 +5,9 @@
       "id": "f-9qhf-bigbear~ca~us",
       "spec": "gtfs",
       "urls": {
-        "static_current": "https://api.transloc.com/gtfs/mountaintransit.zip",
+        "static_current": "https://data.trilliumtransit.com/gtfs/bigbear-ca-us/bigbear-ca-us.zip",
```

Comment:
@westontrillium just confirming that the Trillium URL is the best place to source this feed from currently?

Reply:
Yes, definitely. Thank you for double checking!